terraform/stacks/networking/README.md

```diff
@@ -72,28 +82,30 @@ from stacks.common import CommonStack
 
 app = core.App()
 
-common_props = {
-    'vpc_id': 'vpc-01234567890'  # output of main vpc ID
-}
-
-batch_app_props = {}
+env_dev = core.Environment(account="123456789", region="ap-southeast-2")
```

Comment:
One "principle" we discussed with @reisingerf recently is that we'll start taking things like accountIDs away from (CDK) scripts, fetching them implicitly from the currently authenticated account (on `~/.aws`, parameter store, `cdk.context.json` or similar). I'm not 100% sure if that's easy/doable with this particular example, but just leaving it here as an observation.

Reply:
- Yap, agree. Updated to use those from `~/.aws` i.e. with `cdk synth --profile=dev` [flag from AWS CLI profile](https://docs.aws.amazon.com/cdk/latest/guide/environments.html).
- Please see updated [README.md](https://github.com/umccr/infrastructure/tree/networking-dns-hostname/terraform/stacks/networking) on this.
- Also provided [`debug.py`](https://github.com/umccr/infrastructure/blob/networking-dns-hostname/terraform/stacks/networking/debug.py) sample for CDK try out for querying this main vpc and its subnets tier filtering.